### DISCUSSION TITLE
韓国語でもイシュー報告をうける？

### DISCUSSION THREAD

**Discussion by saschanaz** [misskey-dev/misskey#11424] [General]
韓国語のサーバー管理者やユーザーがMisskeyソフトウェアから問題を見つけても、日本語も英語もできなくて戸惑ってしまうことがあります。今一応私もmisskey-devの一員で、何とか日本語翻訳付きで対応できないかなと考えていますが、どうでしょうか。たぶんこうなるとおもいます

# 韓国語のイシュー

> ## Summary
> 
> 뭔가 한국어로 쓰여 있음
> 
> ## Expected behavior
> 
> 뭔가 한국어
> 
> ## Actual behavior
> 
> 뭔가 한국어

# 私の対応

> 翻訳です
> 
> ## Summary
> 
> 何か韓国語ので書いている
> 
> ## Expected behavior
> 
> 何か韓国語
> 
> ## Actual behavior
> 
> 何か韓国語
>
> ---
> 
> 여기서 뭔가 질문을 함 / ここでなにか質問をする

大体私の翻訳がなくても翻訳機で何とかなると思いますが、韓国語の機械翻訳は大体くずで（）期待できないことが多くて、こうして対応したらよくないかなと思います

**Comment by tai-cha** [comment]
翻訳問題
https://github.com/marketplace/actions/deepl-translate-github-action
こういったもので折りたたみを付けつつBotでIssueの本文の翻訳とかさせられないかな（APIリミットさすがに厳しいか…？）

**Comment by tamaina** [comment]
（GitHub、Issueの翻訳機能と要約機能つけて欲しい）

**Reply by saschanaz** [reply]
（Microsoft, 翻訳機持ってるのにちゃんと使わない）
